### ISSUE TITLE
Infrastructure network throughput is zero when monitor interval ≥ 1 aggregation bucket

### ISSUE THREAD

**Issue by gorkem-bwl** [bluewave-labs/Checkmate#3765]
### Summary

The Infrastructure > Network charts show zero throughput (and the "Capture is reporting zero traffic … container … loopback interfaces" notice) even when Capture is running on a real host and the interface counters are actively growing. This happens because the server computes per-second throughput as `(last check bytes_sent − first check bytes_sent) / (last − first)` **within a single aggregation bucket**, and buckets frequently contain only one sample.

### Details

**Aggregation:** `server/src/domain/checks/check.repository.mongo.ts` — `getHardwareStats` (~lines 657–798) buckets checks by `$dateToString` with a format string like `%Y-%m-%dT%H:%M:00Z` (one bucket per minute for the default `recent` range).

**Bucket format map:** `server/src/domain/monitors/monitor.service.ts:157–166`.

```
recent:  %Y-%m-%dT%H:%M:00Z   (per minute)
day:     %Y-%m-%dT%H:00:00Z   (per hour)
week:    %Y-%m-%dT00:00:00Z   (per day)
month:   %Y-%m-%dT00:00:00Z   (per day)
all:     %Y-%m-%dT00:00:00Z   (per day)
```

**Delta computation (per interface):**

```js
bytesSentPerSecond = {
  $let: {
    vars: {
      tDiff: { $divide: [{ $subtract: [{ $last: "$createdAts" }, { $first: "$createdAts" }] }, 1000] },
      f:     { $arrayElemAt: [{ $map: { input: { $first: "$net" }, as: "i", in: "$$i.bytes_sent" } }, "$$nIdx"] },
      l:     { $arrayElemAt: [{ $map: { input: { $last: "$net" },  as: "i", in: "$$i.bytes_sent" } }, "$$nIdx"] },
    },
    in: { $cond: [{ $gt: ["$$tDiff", 0] }, { $divide: [{ $subtract: ["$$l", "$$f"] }, "$$tDiff"] }, 0] },
  },
}
```

When a bucket contains **one** check, `$first === $last` → `l - f = 0` → `bytesSentPerSecond = 0`. Same for every `deltaX` field.

### Repro

- Monitor interval ≥ 60 s (this is the default).
- Range = "recent" (default, 2-hour view).
- Backend produces per-minute buckets; each bucket has one sample; every interface reports 0 throughput regardless of what the underlying `/proc/net/dev` counters show.

### Impact

- Empty network charts on hosts that clearly have traffic.
- Misleading "container / loopback" notice fires (fixed in the companion PR #3766 to trigger on interface name instead, but the empty charts remain).

### Suggested fix directions

Any one of these solves the root cause; pick whatever fits the ingest path:

1. **Compute per-check delta at ingest.** On each new hardware check, look up the previous check for the same monitor+interface, compute `(bytes_sent_now − bytes_sent_prev) / (t_now − t_prev)` and store the result on the check document. Aggregation then `$avg`s that number and one-sample buckets show real throughput.
2. **Cross-bucket seed sample.** In the aggregation, include the check *immediately preceding* each bucket as the `$first` reference, so a single-sample bucket has two comparison points.
3. **Fall back to instantaneous rate.** When a bucket has one sample and the previous bucket has one sample, compute the delta *across* those two samples in a follow-up $lookup stage.

Option 1 is the cleanest — it moves the compute to write time and keeps the aggregation simple.

### Companion PR

Frontend fix so the misleading notice stops firing on hosts with real NICs: #3766 (renders notice only when the *reported interface names* are all loopback/container-bridge — `lo`, `lo0`, `docker0`).

### Acceptance criteria

- [ ] Charts show real throughput on a host running Capture directly, with the default monitor interval (60 s) and default "recent" range.
- [ ] Delta values are consistent across ranges (recent/day/week/month).
- [ ] Test covers a single-sample bucket producing a non-zero throughput value.

**Comment by kevin-fryskict** [comment]
I also ran into something like this issue. I have checkmate installed on vps 3, and capture installed on a separate VPS (no. 1 and no. 2) with live traffic, everything goes through smoothly (except on vps1 it's not showing a disk graph - it is showing the smaller box with the donut chart, just not the graph, whereas three other servers do also show the disk usage graph - but that's hopefully separate from this). When I changed the monitor frequency from the default 1 minute to 15 seconds, the network graph started showing something. I currently have vps 2 polling every 30 seconds and that apparently also works.

**Comment by gorkem-bwl** [comment]
PR merged.
